packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -1211,6 +1211,50 @@ const POSTOmnichannelContactSchema = {
 
 export const isPOSTOmnichannelContactProps = ajv.compile<POSTOmnichannelContactProps>(POSTOmnichannelContactSchema);
 
+type POSTOmnichannelContactsProps = {
+	name: string;
+	emails: string[];
+	phones: string[];
+	customFields?: Record<string, unknown>;
+	contactManager?: string;
+};
+
+const POSTOmnichannelContactsSchema = {
+	type: 'object',
+	properties: {
```

Comment:
Aren't we forgetting `channels` here? 🤔

Reply:
Actually, no. The channels will be set automatically when we create a contact. We will set the channels based on visitor interaction, and the user cannot manually set this property when creating a contact.